app/screens/About/styles.js

```diff
@@ -1,57 +1,55 @@
-import {StyleSheet,Dimensions} from 'react-native'
-import { Icon } from 'native-base';
-const width = Dimensions.get('window').width;
+import { StyleSheet } from 'react-native'
 import Color from '../../utils/colorConstants'
 
-export const aboutPage =(colorFile, sizeFile) =>{
+export const aboutPage = (colorFile, sizeFile) => {
     return StyleSheet.create({
-        container:{
-            flex:1,
-            backgroundColor:colorFile.backgroundColor,
-        },
-        textStyle:{
-                fontSize:sizeFile.contentText,
-                color:colorFile.textColor,
-                lineHeight:sizeFile.lineHeight,
-                // flexShrink: 1 ,
-        },
-        textContainer:{
-            margin:16,
-            padding:8
-        },
-        featureList:{
-            fontSize:sizeFile.contentText,
-            color:colorFile.textColor,
-            fontWeight:"bold"
-        },
-        italicText:{
-            fontWeight:'bold',fontStyle:'italic'
-        },
-        boldText:{
-            fontWeight:"bold"
-        },
-        linkText:{
-                color: Color.Red,
-                textDecorationLine:'underline',
-                fontSize:sizeFile.contentText
-        },
-        featureView:{
-            flexDirection:'row'
-        },
-        TitleText:{
-            paddingVertical:12,
-            fontSize:sizeFile.titleText,
+        container: {
+            flex: 1,
+            backgroundColor: colorFile.backgroundColor,
+        },
+        textStyle: {
+            fontSize: sizeFile.contentText,
+            color: colorFile.textColor,
+            lineHeight: sizeFile.lineHeight,
+            // flexShrink: 1 ,
```

Comment:
lot of commented lines in this file. Check and remove if unnecessary

Reply:
ok,I'll